stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -62,6 +62,9 @@ spec:
       {{- if .Values.master.affinity }}
       affinity:
 {{ toYaml .Values.master.affinity | indent 8 }}
+      {{- end }}
+      {{- if quote .Values.master.terminationGracePeriodSeconds }}
```

Comment:
Why do you use quote?

Reply:
> Can you add an entry in changeling?

Done

Reply:
> Why do you use quote?

While testing with value 0, the value was not inserted.  It turns out that without the `quote` the if statement interprets the 0 value as False and it's not inserted.

Reply:
Thanks for the explanation.